primitives/src/vstaging/executor_params.rs

```diff
@@ -92,6 +112,16 @@ impl ExecutorParams {
 	pub fn hash(&self) -> ExecutorParamsHash {
 		ExecutorParamsHash(BlakeTwo256::hash(&self.encode()))
 	}
+
+	timeout_by_kind! {
```

Comment:
Not a fan of macro usage here: Obfuscates the interface, would definitely prefer a bit of code duplication here over a macro.